arm/Microsoft.RecoveryServices/vaults/deploy.bicep

```diff
@@ -239,6 +239,7 @@ module rsv_backupConfig 'backupConfig/deploy.bicep' = if (!empty(backupConfig))
     storageTypeState: contains(backupConfig, 'storageTypeState') ? backupConfig.storageTypeState : 'Locked'
     isSoftDeleteFeatureStateEditable: contains(backupConfig, 'isSoftDeleteFeatureStateEditable') ? backupConfig.isSoftDeleteFeatureStateEditable : true
     enableDefaultTelemetry: enableChildTelemetry
```

Comment:
Please remove this line as it has been correctly replaced below
```suggestion
```

Reply:
Done